Documentation/gettingstarted/k8s-install-azure-cni-steps.rst

```diff
@@ -73,6 +66,7 @@ Generate the required YAML file and deploy it:
      --set global.cni.chainingMode=generic-veth \
      --set global.cni.customConf=true \
      --set global.nodeinit.enabled=true \
+     --set nodeinit.azure=true \\
```

Comment:
Is the double slash intentional?

Reply:
Good catch, not intentional. The original commit needed this but when rebasing I should have dropped it.